addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -673,6 +673,16 @@ def _import_fattura_pa_line(self, element, invoice_line_form, extra_info=None):
         if line_elements:
             invoice_line_form.sequence = int(line_elements[0].text)
 
+        # Label.
+        line_elements = element.xpath('.//Descrizione')
+        if line_elements:
+            invoice_line_form.name = " ".join(line_elements[0].text.split())
+
+        # Fitting account for the line
+        fitting_account_id = invoice_line_form.find_fit_account()
```

Comment:
```suggestion
        fitting_account = invoice_line_form.find_fit_account()
```

It's not an id; it's a record. If you call it fitting_account_id, you're implying it's an int. This error is made by a lot of people, because guidelines tell you to call your relational fields ***_id ; but here, it's not a field. For fields, we do that because they end up being stored as columns in the database, in which the id is stored ; so it matches the SQL standards.